spec/domain/invoices/sac_memberships/position_generator_spec.rb

```diff
@@ -606,9 +606,7 @@
 
     context "last year without reduction" do
       before do
-        # because days are divided by 365, people with an entry date before january 9th
-        # will have 25 years already one year earlier
-        roles(:mitglied).update!(created_at: Date.new(date.year - 24, 1, 9))
+        roles(:mitglied).update!(created_at: Date.new(date.year - 24, 1, 1))
       end
 
       it "generates positions without reduction" do
```

Comment:
for some reason this spec still fails, even tho when I check the membership years 

```
pry(#<RSpec::ExampleGroups::InvoicesSacMembershipsPositionGenerator::WithSectionMembershipYearsReduction::LastYearWithoutReduction>)> Person.with_membership_years.find(people(:mitglied).id).membership_years
=> 24
```
they should be correct, because reduction starts from 25 years...